receiver/aerospikereceiver/config.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aerospikereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver"
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver/internal/metadata"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configtls"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+	"go.uber.org/multierr"
+)
+
+var (
+	defaultEndpoint              = "localhost:3000"
+	defaultTimeout               = 20 * time.Second
+	defaultCollectClusterMetrics = false
+)
+
+var (
+	errBadEndpoint   = errors.New("endpoint must be specified as host:port")
+	errBadPort       = errors.New("invalid port in endpoint")
+	errEmptyEndpoint = errors.New("endpoint must be specified")
+	errEmptyPassword = errors.New("password must be set if username is set")
+	errEmptyUsername = errors.New("username must be set if password is set")
+)
+
+// Config is the receiver configuration
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+	Endpoint                                string                     `mapstructure:"endpoint"`
+	Username                                string                     `mapstructure:"username"`
+	Password                                string                     `mapstructure:"password"`
```

Comment:
I'm having trouble find where these are used to authenticate with aerospike; Are these currently being used?